core/sql/sqlcomp/CmpSeabaseDDLschema.cpp

```diff
@@ -1006,7 +1006,7 @@ void CmpSeabaseDDL::alterSeabaseSchema(StmtDDLAlterSchema * alterSchemaNode)
 
    bool isVolatile = (memcmp(schName.data(),"VOLATILE_SCHEMA",strlen("VOLATILE_SCHEMA")) == 0);
    int32_t length = 0;
-   int32_t rowCount = 0;
+   Int64 rowCount = 0;
```

Comment:
It doesn't look like this particular "rowCount" variable is used anywhere. I suppose we could delete it. The code change is harmless though.